docs/content/preview/yugabyte-voyager/reference/data-migration/import-data.md

```diff
@@ -210,6 +210,14 @@ send-diagnostics:
 
 | Enable or disable sending [diagnostics](../../../reference/diagnostics-report/) information to Yugabyte. <br>Default: true<br> Accepted parameters: true, false, yes, no, 0, 1 |
 
+| --max-retries-streaming |
+
+```yaml{.nocopy}
+max-retries-streaming:
```

Comment:
it should be inside the command key,
e.g. 
```
import-data:
    max-retries-streaming:
```

same for other import data to source/source-replica ones as well